src/ZODB/MappingStorage.py

```diff
@@ -232,7 +233,7 @@ def sortKey(self):
 
     # ZODB.interfaces.IStorage
     @ZODB.utils.locked(opened)
-    def store(self, oid, serial, data, version, transaction):
+    def store(self, oid, serial, data, version, transaction, resolved=False):
```

Comment:
> `MappingStorage` doesn't and should have `_resolved`.

Do you mean it shouldn't ?

I'll move all `_resolved` in  `MappingStorage` to `DemoStorage`.